front/src/modules/ui/input/relation-picker/components/SingleEntitySelectBase.tsx

```diff
@@ -30,10 +33,12 @@ export function SingleEntitySelectBase<
   entities,
   onEntitySelected,
   onCancel,
+  noUser,
 }: {
   entities: EntitiesForSingleEntitySelect<CustomEntityForSelect>;
   onEntitySelected: (entity: CustomEntityForSelect | null | undefined) => void;
   onCancel?: () => void;
+  noUser?: CustomEntityForSelect;
```

Comment:
let's pass a noUserPlaceholder { icon: string, label: string } instead to avoid hacking into CustomEntityForSelect

Reply:
Sounds good!